### ISSUE TITLE
Oil-drops

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10162]
**Preferred term label:**

**Synonyms**
spot-like appearance pigment deposits in the limbus

**Definition (free text, please give PubMed ID)**
PMID: 24479547 (...)  The most frequent sign was symmetric brown sclera pigmentation present in 82.5 percent of the patients. **“Oil-drops”**, brown pigment spots in the limbus are generally considered pathognomonic but were a little less frequent (75 percent)
Pigment may also accumulate in the opaque portion of the limbus where it forms “oil drop”-like brown spots (Figure 2, right, with permission [31]), seen in 30 cases.
Due to their characteristic appearance, corneal **“oil-drops”** are of high diagnostic significance. In agreement with others [10,18], we think that “oil-drops” are pathognomonic to ochronosis.

PMID: 35015405 Another standard (75%) ocular finding that is considered pathognomonic for the disease is the appearance of “oil-drops,” resulting from pigment deposited with a spot-like appearance in the limbus. Pigment deposition in the conjunctiva creates “vermiform” or “tube-like” conjunctival hyperpigmentation. This finding is often accompanied by increased conjunctival vessel diameter due to dilated vasculature supplying areas of pigmentation. These various findings can be summarized by four main appearances of hyperpigmentation which can occur in isolation or combination: vermiform, pinguecula-like, dot-like, and laminar.

**Parent term (use hpo.jax.org/app)**
Abnormal corneal limbus morphology HP:0025348

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:56 Alkaptonuria

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000027
- label: Oil-drop brown pigmentation of the corneal limbus
- parents: [Pigmentation of the sclera (HP:0007832)](https://hpo.jax.org/app/browse/term/HP:0007832)
- pmid: PMID:24479547
- def: Brown pigment spots in the limbus (border between the cornea and the sclera).
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10162
- synonyms:
